recipes/amplisim/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+ls -l -a
+cd lib
+git clone --recursive https://github.com/samtools/htslib.git
+cd htslib
+git reset --hard 7f5136f
+cd htscodecs
+git reset --hard 11b5007
+cd ../../..
```

Comment:
Not a fan of this section -- ideally we could build against the conda-packaged htslib. I think you'd need to patch the makefile.

That said, it's clearly working so I'll accept things now in the interest of getting this into conda, but at some point please modify your upstream makefile to allow using system htslib, and then disable all of this.

Reply:
I agree it's a brutal fix. I'll leave myself an issue in amplisim to fix this in an upcoming release 👍